packages/tokens/src/global/colors.ts

```diff
@@ -1,116 +1,32 @@
-/**
- * @module colors
- * @desc Color tokens module.
- *
- * @memberof @gympass/yoga-tokens
- */
+// Colors ↓
 
-/**
- * A color
- * @typedef Color
- *
- * @type {String}
- */
-
-/**
- * @type {Color}
- */
 const vibin = '#D8385E';
-
-/**
- * @type {Color}
- */
 const hope = '#1D856C';
-
-/**
- * @type {Color}
- */
 const energy = '#FFC24C';
-
-/**
- * @type {Color}
- */
 const relax = '#5340C9';
 
 // Brand colors ↓
 
-/**
- * @type {Color}
- */
 const peace = '#FFB0A7';
-
-/**
- * @type {Color}
- */
 const verve = '#CA4808';
-
-/**
- * @type {Color}
- */
 const brandingVerve = '#FF874C';
-
-/**
- * @type {Color}
- */
 const uplift = '#FF4438';
-
-/**
- * @type {Color}
- */
 const deepPurple = '#710252';
-
-/**
- * @type {Color}
- */
 const yoga = '#FFEEF2';
 
 // Feedback ↓
 
-/**
- * @type {Color}
- */
 const success = '#C1EEDB';
-
-/**
- * @type {Color}
- */
 const neutral = '#E0DFFF';
-
-/**
- * @type {Color}
- */
 const attention = '#FCD6C3';
 
 // Gray scale ↓
 
-/**
- * @type {Color}
- */
 const stamina = '#231B22';
-
-/**
- * @type {Color}
- */
 const deep = '#6B6B78';
-
-/**
- * @type {Color}
- */
 const medium = '#9898A6';
-
-/**
- * @type {Color}
- */
 const light = '#D7D7E0';
-
-/**
- * @type {Color}
- */
 const clear = '#F5F5FA';
-
-/**
- * @type {Color}
- */
 const white = '#FFFFFF';
 
 const colors = {
```

Comment:
```ts
export default colors = {
  vibin,
  hope,
  energy,
  relax,
  peace,
  brandingVerve,
  verve,
  uplift,
  deepPurple,
  yoga,
  success,
  neutral,
  attention,
  stamina,
  deep,
  medium,
  light,
  clear,
  white,
} as const;
```

Reply:
For the future, not for now, we should probably avoid exporting default. we're polluting the bundle with all of the colors if we only use one

(named exports scale a lot better than default exports :P)